packages/obonode/obojobo-chunks-multiple-choice-assessment/viewer-component.js

```diff
@@ -494,9 +494,7 @@ export default class MCAssessment extends React.Component {
 			>
 				<fieldset>
 					<legend className="instructions">
-						<span className="for-screen-reader-only">{`Multiple choice form with ${
-							sortedChoiceModels.length
-						} choices. `}</span>
+						<span className="for-screen-reader-only">{`Multiple choice form with ${sortedChoiceModels.length} choices. `}</span>
```

Comment:
We need to change these instructions since this isn't a multiple choice form

Reply:
This is actually in MCAssessment. I was accidentally formatted it using Prettier.